packages/tools/src/tools/segmentation/BrushTool.ts

```diff
@@ -76,41 +106,70 @@ class BrushTool extends BaseTool {
           THRESHOLD_INSIDE_SPHERE: thresholdInsideSphere,
         },
         strategySpecificConfiguration: {
-          THRESHOLD_INSIDE_CIRCLE: {
+          THRESHOLD: {
             threshold: [-150, -70], // E.g. CT Fat // Only used during threshold strategies.
           },
         },
         defaultStrategy: 'FILL_INSIDE_CIRCLE',
         activeStrategy: 'FILL_INSIDE_CIRCLE',
         brushSize: 25,
+        preview: {
+          // Have to enable the preview to use this
+          enabled: false,
+          previewColors: {},
+          // The time before showing a preview
+          previewTimeMs: 250,
+          // The distance to move to show a preview before preview time expired
+          previewMoveDistance: 8,
+          // The distance to drag before being considered a drag rather than click
+          dragMoveDistance: 4,
+          // The time to consider a mouse click a drag when moved less than dragMoveDistance
+          dragTimeMs: 500,
```

Comment:
seems like you still need this?

Reply:
Yes, but it is only used for preview.  Otherwise you can end up with a different selection than you intended.  I can demonstrate if we set the preview time/drag times to fairly large values so it is easy to show how you get the wrong final selection.